index.html

```diff
@@ -11,18 +11,32 @@
   <body>
       <!-- simple layout  *change to suit needs -->
       <div class="container">
```

Comment:
@elementWebDev for the navbar, could you fix where the .container divs are. I think they are slightly out of place.

Reply:
Sounds good. Sorry about that.. I will set it to the master